packages/typespec-vscode/src/telemetry/telemetry-client.ts

```diff
@@ -42,12 +43,8 @@ export class TelemetryClient {
       );
       this._client = undefined;
     } else {
-      this._client = new (TelemetryReporter as any)(
-        `${pkgJson.publisher}.${pkgJson.name}`,
```

Comment:
are those info not pass in any other way now?

Reply:
the latest vscode telemetry will collect these inforamtion automatically, so dont need to pass them explicitly.thx